package.json

```diff
@@ -17,7 +17,7 @@
     "optimist": "^0.6.1",
     "xhr": "^2.5.0"
   },
-  "bin": "./bin/dogescript.js",
+  "bin": "./bin/dogescript.bin.js",
```

Comment:
any reason to not use the `dist/` based file? I believe webpack adds a shebang for a node environment. ?

Reply:
I think doing that would mean we need a prepare script that runs the build

Reply:
I guess then based on the definition for `bin` from here: https://docs.npmjs.com/files/package.json

We probably should just copy the shebang line and not use webpack: https://github.com/dogescript/dogescript/blob/master/webpack.config.js#L36

Or we should use a prepublish/prepare script then to run webpack and get that shebang added.

Reply:
@Pholey should we add the shebang to this file then?